### ISSUE TITLE
One way of using VoiceOver does not allow user to move masses

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/gravity-force-lab#202]
**Test Device**

MacBook Air

**Operating System**

macOS 10.14.6

**Browser**

Safari 13.0.3

**Problem Description**

For https://github.com/phetsims/QA/issues/455.

There is a scenario in which a user is not able to move a mass with the keyboard. I'm wondering if the way in which VoiceOver and keyboard navigation are used in this scenario is standard practice.

**Steps to Reproduce**

1. `control` + `option` + `shift` + `down arrow` to enter web content.
2. Navigate your way to one of the masses.
3. Make sure VoiceOver is instructing you to "press `control` + `option` + `shift` + `down arrow`" to interact with the mass's slider.
4. Press `control` + `option` + `shift` + `down arrow` so that you are interacting with the mass's slider.
5. At this point, the user is trying to move the mass. There are two scenarios:
    1. The only keys used to move the mass are the arrow keys.
    2. The keys used to move the mass are `control` + `option` + an arrow key.

* In scenario i, the mass moves as expected.
* In scenario ii, the mass does not move. The slider in the PDOM moves.

Is the way in which VoiceOver and keyboard navigation are used in scenario ii standard practice? In other words, if a user is interacting with a slider on macOS with VoiceOver, do you expect them to use `control` + `option` + an arrow key?

I know that our Keyboard Shortcuts dialogue instructs the user to use the arrow keys to change the mass, but it would seem that the default behavior for macOS VoiceOver keyboard interaction with a slider is as follows:

1. You keyboard navigate to the slider.
2. You press `control` + `option` + `shift` + `down arrow` to interact with the slider.
3. Only using an arrow key adjusts the slider in steps of size 1 and using `control` + `option` + an arrow key adjusts the slider in steps of size 5.

See https://jsfiddle.net/LiamEM/v9y5du06/ if you want to review the default behavior.

If a user is expecting to be able to move the mass by adjusting the slider as in scenario ii, they won't be able to move the mass; they'll be adjusting the slider in the PDOM.

**Visuals**

This is scenario i followed by scenario ii:
![issue](https://user-images.githubusercontent.com/31285590/68815086-adf98400-0637-11ea-8e8b-785ab28ba5b1.gif)

<details>
<summary><b>Troubleshooting information</b></summary>
!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.16/phet/gravity-force-lab_all_phet.html
Version: 2.2.0-dev.16 2019-11-06 21:45:42 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.87 Safari/537.36
Language: en-US
Window: 1440x821
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by terracoda** [comment]
@lmulhall-phet, I don't seem to be experiencing the same issue.
- the slider does moves visually - same as you, and when I use control + option + an arrow the pdom does not seem to change in value. At least, the no changed value is read out by VO.
- I don't see the little square black VO highlight that you see, so I can't tell if the slider actually moves, but the descriptions that VO announces do not change 
- Also, more importantly, I never hear instructions from VO to use  control + option + an arrow key to adjust the slider value. Are your VO hints set to verbose? I actually turn them off, but I turned them on for this test, and I never hear VO tell me to use control + option + an arrow key.

I don't think we have to implement VO specific key presses for the following reasons:
- we have general shortcuts built-in on sliders for regular steps (Arrow keys), smaller steps (our custom modifier key, Shift + Arrow keys), and bigger steps (Pg Up/Pg Down), and Home/End keys to jump to min and max values.
- I have never had a VO user complain about our approach. They have used the general (and the custom small steps) shortcuts with ease and without complaint.
- I think our general shortcuts recommended by W3C standards seem to satisfy their needs.

**Comment by terracoda** [comment]
@lmulhall-phet, I am using VO in MacBook Pro MacOS 10.13.6

**Comment by terracoda** [comment]
Personally, I wuold mark this as Screen Reader specific and will not fix. 
@lmulhall-phet, does that make sense to you, or did I miss something?

**Comment by terracoda** [comment]
I will add that this bug also exists in published version of RIAW, and we might have decided not to fix it in the QA of that sim. I'm not sure how to look for that.

**Comment by terracoda** [comment]
@lmulhall-phet, please reassign to me if you think I missed something.

**Comment by terracoda** [comment]
@lmulhall-phet, and I realized having the black VO cursor highlight is a VO setting. When I turn it on, I see what you see.

**Comment by lmulhall-phet** [comment]
@terracoda, as long as you don't think someone would find this confusing, then I think it's OK to close.

Just to be clear, VoiceOver doesn't instruct me to use `control` + `option` + an arrow key to move the slider, I just happened to do it that way. That's how I discovered scenario ii. I wasn't sure if this was standard practice or just a weird thing I was doing.

Thanks for looking into this, @terracoda. Sorry that it was a non-issue. Please close if you also think it's OK to close.

**Comment by terracoda** [comment]
@lmulhall-phet, I appreciate your testing a lot, so thanks. We don't support every screen reader command, though is some critical cases we can chose to do something that is screen reader specific.

Screen reader commands are highly customizable, so following W3C ARIA interaction patterns is a safer bet in general.

Thanks for your careful testing. Closing.
